includes/class-wcpdf-admin.php

```diff
@@ -1,22 +1,34 @@
 <?php
+
 namespace WPO\WC\PDF_Invoices;
 
+use \Automattic\WooCommerce\Internal\DataStores\Orders\CustomOrdersTableController;
+
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly
 }
 
-if ( !class_exists( '\\WPO\\WC\\PDF_Invoices\\Admin' ) ) :
+if ( ! class_exists( '\\WPO\\WC\\PDF_Invoices\\Admin' ) ) :
 
 class Admin {
+
 	function __construct()	{
 		add_action( 'woocommerce_admin_order_actions_end', array( $this, 'add_listing_actions' ) );
 
-		add_filter( 'manage_edit-shop_order_columns', array( $this, 'add_invoice_columns' ), 999 );
-		add_action( 'manage_shop_order_posts_custom_column', array( $this, 'invoice_columns_data' ), 2 );
-		add_filter( 'manage_edit-shop_order_sortable_columns', array( $this, 'invoice_columns_sortable' ) );
+		if ( defined( 'WC_VERSION' ) && version_compare( WC_VERSION, '7.1', '>=' ) ) {
```

Comment:
@alexmigf Why is the condition and branching needed? Presumably the actions/filters are only called when needed, so they don't need to be conditional.